### PR TITLE
fix: Task package tag

### PR DESCRIPTION
**Pull request by nmimsnikhil**
[Asana](https://app.asana.com/0/1192407899754728/1199386726415517)
image

### CODE REVIEW

**Review by rvpasari — DISMISSED** [review]

File: erpnext/patches/v13_0/update_global_search.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import frappe
+from frappe.desk.page.setup_wizard.install_fixtures import update_global_search_doctypes
+
+def execute():
+    frappe.reload_doc("desk", "doctype", "global_search_doctype")
+    frappe.reload_doc("desk", "doctype", "global_search_settings")
```
Comment (hrwX) [inline erpnext/patches/v13_0/update_global_search.py]:
```suggestion
```

File: erpnext/patches/v13_0/update_global_search.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    update_global_search_doctypes()
```
Comment (hrwX) [inline erpnext/patches/v13_0/update_global_search.py]: - add `Package Tag` using get_doc

File: erpnext/hooks.py [hunk 3]
```diff
@@ -445,6 +445,7 @@
 		{"doctype": "Maintenance Schedule", "index": 44},
 		{"doctype": "Maintenance Visit", "index": 45},
 		{"doctype": "Warranty Claim", "index": 46},
+		{"doctype": "Package Tag", "index": 47},
```
Comment (hrwX) [inline erpnext/hooks.py]: Add this above Pick List

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/patches/v13_0/update_global_search.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    for i in doc.allowed_in_global_search:
+        idx += 1
+        if idx == 18:
+            idx += 1
+        i.idx = idx
+    doc.append("allowed_in_global_search", {
+        "document_type": "Package Tag",
+        "idx":  18,
+        })
+    doc.save()
```
Comment (hrwX) [inline erpnext/patches/v13_0/update_global_search.py]: doc.allowed_in_global_search.insert(18, {
	"document_type": "Package Tag",
	"idx":  18
})
doc.save()

**Review by hrwX — CHANGES_REQUESTED** [review]

**Review by hrwX — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**